### PR TITLE
Feedback webscreenlet

### PR DESCRIPTION
**Pull request by victorg1991**
This PR fix this problems

- Fix ssl errors
- Remove comments in files allowing developers to use //coments in the js code.

### CODE REVIEW

File: android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/WebActivity.java [hunk 1]
```diff
@@ -33,6 +34,7 @@ protected void onCreate(Bundle savedInstanceState) {
 					R.raw.portletcss, "portlet.css")
 					.addLocalCss("gallery.css")
 					.addLocalJs("gallery.js")
+
```
Comment (jmnavarro) [inline android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/WebActivity.java]: empty line?

File: android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java [hunk 2]
```diff
@@ -55,7 +55,9 @@ private String getFileContent(InputStream inputStream) throws Exception {
 		String line = reader.readLine();
 
 		while (line != null) {
-			fileContent.append(line);
+			if (!line.startsWith("//")) {
```
Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java]: What if I use this method to read a plain text file?

Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java]: The name is maybe not the best, but we are using this class to read js and css files only :(

File: android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/WebActivity.java (hunk 1 above)
Comment (victorg1991) [reply android/samples/test-app/src/main/java/com/liferay/mobile/screens/testapp/WebActivity.java]: 👍

File: android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java (hunk 2 above)
Comment (jmnavarro) [reply android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java]: then, renamed the method to something more specific

Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java]: 👍

**Comment by victorg1991** [comment]
I thinks is ready to go 🚀

**Pull request merged**